tests/vmcheck/test.sh

```diff
@@ -56,7 +56,7 @@ vm_cmd ostree refs vmcheck_tmp vmcheck_remote --delete
 # comes with the distro to help speed up rpm-ostree metadata fetching since we
 # don't cache it (e.g. on Fedora, it takes *forever* to fetch metadata, which we
 # have to do dozens of times throughout the suite)
-if ! vm_cmd test -f /etc/yum.repos.d/.vmcheck; then
+if vm_cmd test -d /etc/yum.repos.d/ && ! vm_cmd test -f /etc/yum.repos.d/.vmcheck; then
```

Comment:
How about just doing:

```
vm_cmd mkdir -p /etc/yum.repos.d/
```

upfront instead? It should be fine to leave an empty `/etc/yum.repos.d/` around afterwards.

Reply:
Oh man...github says you won by *16 seconds*, I'll have to live with :2nd_place_medal: